hardware_layer/hardware_arduino/src/MS5837.h

```diff
@@ -1,10 +1,10 @@
 /* Blue Robotics Arduino MS5837-30BA Pressure/Temperature Sensor Library
```

Comment:
this files were to be left untouched :/ thats why I modified the roslint in Cmake to not check these for lint.